ftdc/custom_format.go

```diff
@@ -337,6 +339,15 @@ type Reading struct {
 	Value      float32
 }
 
+// ConvertedTime turns the `Time` int64 value in nanoseconds since the epoch into a `time.Time`
+// object in the UTC timezone.
+func (flatDatum *FlatDatum) ConvertedTime() time.Time {
```

Comment:
I had a bigger change that updated datum time members to be a `time.Time` instead of an `int64`. Maybe still worth doing, but not for right now.